### PR TITLE
Tiny fixes for the documentation

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

Nothing changes. Just some tiny fixes for the documentation.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -85,9 +85,6 @@ Contacting Us
   open issue or pull request.
 * We have a `Discourse forum <https://forum.generic-mapping-tools.org>`__
   where you can ask questions and leave comments.
-* This project is released with a `Contributor Code of Conduct
-  <https://github.com/GenericMappingTools/pygmt/blob/master/CODE_OF_CONDUCT.md>`__.
-  By participating in this project you agree to abide by its terms.
```
Comment (seisman) [inline README.rst:90]: This sentence can be found at line 99 in the same file.

File: examples/tutorials/first-figure.py [hunk 2]
```diff
@@ -66,8 +66,7 @@
 #
 # You’ll probably have noticed several things that are different from classic
 # command-line GMT. Many of these changes reflect the new GMT modern execution mode that
-# will be part of the future 6.0 release. A few are PyGMT exclusive (like the
-# ``savefig`` method).
+# are part of the GMT 6. A few are PyGMT exclusive (like the ``savefig`` method).
```
Comment (weiji14) [inline examples/tutorials/first-figure.py:69]:
```suggestion
# are part of GMT 6. A few are PyGMT exclusive (like the ``savefig`` method).
```

**Review by weiji14 — COMMENTED** [review]
Just one minor suggestion, otherwise all good.

As an aside, I was thinking of improving our front page badges:

- Add a Binder badge (or a link to [try-gmt](https://github.com/GenericMappingTools/try-gmt)) so that users can try PyGMT directly without installing.
- Maybe remove Codacy and Travis CI Linux badge since they're not particularly useful, and this would mean all our badges fit on one line.

**Comment by seisman** [comment]
> * Add a Binder badge (or a link to [try-gmt](https://github.com/GenericMappingTools/try-gmt)) so that users can try PyGMT directly without installing.

Yes, note that the badges are not shown on the PyGMT website. Perhaps we should also add a badge in the "Installation page"(https://www.pygmt.org/dev/install.html).

> * Maybe remove Codacy and Travis CI Linux badge since they're not particularly useful, and this would mean all our badges fit on one line.

We're still relying on Travis CI to deploy the documentation. So let's still keep it until we fully remove Travis.

As for Codacy, I have no idea what it is. Never used it.

**Review by weiji14 — APPROVED** [review]
> Yes, note that the badges are not shown on the PyGMT website. Perhaps we should also add a badge in the "Installation page"(https://www.pygmt.org/dev/install.html).

Yeah, I always wondered why that was the case. Would still prefer it being on the front page rather than being hidden in Installation. Maybe add a text link "Try-Online" (or other wording) instead of a badge so it will show up?

> We're still relying on Travis CI to deploy the documentation. So let's still keep it until we fully remove Travis.

Ok, wait for #566 then.

> As for Codacy, I have no idea what it is. Never used it.

It's a code quality service added in #34, along with codecov and codeclimate (which we should remove too). we can remove this in a separate PR.

**Comment by seisman** [comment]
> > Yes, note that the badges are not shown on the PyGMT website. Perhaps we should also add a badge in the "Installation page"(https://www.pygmt.org/dev/install.html).
> 
> Yeah, I always wondered why that was the case. Would still prefer it being on the front page rather than being hidden in Installation. Maybe add a text link "Try-Online" (or other wording) instead of a badge so it will show up?

There is a line `.. placeholder-for-doc-index` in the README.rst. Only things after this one are shown in the website.

**Pull request merged**